src/components/date-picker/date-picker.tsx

```diff
@@ -46,6 +46,9 @@ export class DatePicker implements LocalizedComponent {
   //
   //--------------------------------------------------------------------------
 
+  /** Active date */
```

Comment:
Yes, this is needed because it shows what the value would be in the calendar pre-commit, so it highlights it in gray before the value is committed, at which point it turns to blue to denote it is selected.